core/domain/exp_services.py

```diff
@@ -813,15 +1034,28 @@ def update_exploration(
 
 
 def create_exploration_summary(exploration_id, contributor_id_to_add):
-    """Create summary of an exploration and store in datastore."""
+    """Create summary of an exploration and store in datastore.
+
+    Args:
+        exploration_id: str. The id of the exploration.
+        contributor_id_to_add: str or None. ID of the user to be added to
+            the contributor to the exploration.
```

Comment:
Not sure what "the contributor to the exploration" means. What list is this user_id being added to, exactly?

Also, what if this value is None?